src/main/java/io/lettuce/core/pubsub/RedisPubSubReactiveCommandsImpl.java

```diff
@@ -158,6 +159,15 @@ public Mono<Map<K, Long>> pubsubShardNumsub(K... shardChannels) {
         return createMono(() -> commandBuilder.pubsubShardNumsub(shardChannels));
     }
 
+    @Override
+    public Mono<Void> ssubscribe(K... shardChannels) {
+        return createFlux(() -> {
+            Command<K, V, V> c = commandBuilder.ssubscribe(shardChannels);
+            System.out.println("");
```

Comment:
this doesn't look right with the sysout in place.

Reply:
done